python-backend/app/api/mines/variances/resources/variance_list.py

```diff
@@ -2,19 +2,20 @@
 from flask_restplus import Resource, fields
 from werkzeug.exceptions import BadRequest, NotFound
 
-from ..models.variance import Variance
 from ....utils.access_decorators import requires_any_of, MINE_VIEW, MINE_CREATE, MINESPACE_PROPONENT
 from ...mine.models.mine import Mine
 from ....utils.resources_mixins import UserMixin, ErrorMixin
 from app.api.utils.custom_reqparser import CustomReqparser
 from app.api.mines.mine_api_models import VARIANCE_MODEL
+from app.api.variances.models.variance import Variance
```

Comment:
Why are there two conventions for imports?  '...' vs "app.api.etc".  Is this part of the microservice-afication?

Reply:
It doesn't look like there's any pattern that's been followed when deciding between absolute vs. relative imports. Not sure it's been discussed (and we don't have a linter monitoring this sort of thing).

I chose an absolute import here because `Variance` is outside of the `app/api/mines` directory, and anything over 2-3 levels up is probably too many directories to traverse with relative imports. But that's just my personal opinion.